src/localization/package.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>localization</name>
+  <version>0.0.0</version>
+  <description>The localization package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="jorge@todo.todo">jorge</maintainer>
```

Comment:
@jormunrod Add your real github email please and the content to "Jorge Muñoz". Better for future reference :smile: